src/services/movieServices.js

```diff
@@ -0,0 +1,48 @@
+import { URLS } from './api';
+import * as actions from '../store/actions';
+
+export function getMovies(dispatch, queryParams) {
+    const url = URLS.getUrl(URLS.MOVIES_URL, null, queryParams);
+    return fetch(url).then(data => {
+        return data.json();
+    }).then((data) => {
+        dispatch(actions.getMoviesAction({ movies:data }));
+    }, () => {
+        dispatch(actions.getMoviesAction({ movies:{} }));
+    });
+}
+
+
+export function editMovie(dispatch, payload) {
+    const url = URLS.getUrl(URLS.MOVIES_URL, null, null);
+    return fetch(url, {method:'PUT', headers:{'Content-Type': 'application/json'}, body:JSON.stringify(payload)}).then(data => {
```

Comment:
pls transfer each **then** to the new line, for best reading code)
in the whole file)